### DISCUSSION TITLE
An online database for OU VBs

### DISCUSSION THREAD

**Discussion by shuntia** [stakira/OpenUtau#1139] [Ideas]
Many people are having a hard time trying to find and/or share their Utau(and others like DiffS, Enunu) banks. It would improve the experience for many people if there is a function like Singers>browse.
Artists could filter with tags set by the creator, and creators can spread the information(via OU or the browser) of their voicebanks easier.
There may be problems with duplicates, but as an idea, you could make a reporting system for duplicates, or voting systems for tags.

**Comment by Nexys4t** [comment]
I wonder if "many people" really is that many people.

Places like UTAU wikis and YouTube exist. I don't think they have a "hard time", they just don't know how to Google. /Gen

**Comment by MaximusWolfie** [comment]
VocaDB (bleh) also exists, you can usually find vocals there.

**Comment by BagelHero** [comment]
It's best to look on sites like youtube, nicodouga, soundcloud, etc for release videos. While a hub for OU compatible banks would be very cool, it's outside of the scope of OpenUtau itself. Perhaps it might make a fun wiki/documentation project, or even a plugin; it's just not something that can be a part of the program as maintained on the main branch, it's too high maintenance.

**Comment by oxygen-dioxide** [comment]
github.com/openutau/svs-index
